animations/lib/src/basics/08_fade_transition.dart

```diff
@@ -4,6 +4,10 @@
 
 import 'package:flutter/material.dart';
 
+/* 
+Refer to the AnimatedWidget docs here - https://api.flutter.dev/flutter/widgets/AnimatedWidget-class.html
+for examples of other common animated widgets. 
+*/
```

Comment:
Consider changing this to a `//` comment, it's considered better style to avoid block comments for documentation: https://dart.dev/guides/language/effective-dart/documentation#dont-use-block-comments-for-documentation